### ISSUE TITLE
Cors Error while Adding Blog

### ISSUE THREAD

**Issue by ojasaklechayt** [ojasaklechayt/Oju-s-Blogs#28]
Checkout #24 
While uploading the new blog, we are facing the cors error.
![image](https://github.com/ojasaklechayt/Oju-s-Blogs/assets/90605717/8b3b8348-54c2-43c1-80d2-9bc7c6b2ab49)

**Comment by ojasaklechayt** [comment]
@aasimtaif guys while sending the post request, I am getting this errors. Can you please check because, since you have worked with this, you might have better idea with it.

**Comment by aasimtaif** [comment]
sorry I do not understand the error
[Desktop-1698084965639.webm](https://github.com/ojasaklechayt/Oju-s-Blogs/assets/83891519/f8622891-7588-4ddb-aebc-af62376aabb7)
As you can see in video it is working perfectly fine

**Comment by aasimtaif** [comment]
> Checkout #24 While uploading the new blog, we are facing the cors error. ![image](https://user-images.githubusercontent.com/90605717/277314528-8b3b8348-54c2-43c1-80d2-9bc7c6b2ab49.png)

When I tried this problem did not happen during  blog upload but during the blog edit

**Comment by ojasaklechayt** [comment]
> > Checkout #24 While uploading the new blog, we are facing the cors error. ![image](https://user-images.githubusercontent.com/90605717/277314528-8b3b8348-54c2-43c1-80d2-9bc7c6b2ab49.png)
> 
> When I tried this problem did not happen during blog upload but during the blog edit

This I feel is the cors error which comes when using external APIs. I have already added the cors property in the backend but don't know which it still is coming.

**Comment by aasimtaif** [comment]
This happened to you during blog upload ??

**Comment by ojasaklechayt** [comment]
yep @aasimtaif I have also tried updating this code but this is still happening

**Comment by aasimtaif** [comment]
It is not happening for me can you show the network tab  to see response from backend

**Comment by ojasaklechayt** [comment]
> It is not happening for me can you show the network tab to see response from backend

It give internal server error - 500 to me

**Comment by aasimtaif** [comment]
Then I think it a backend problem
I will look into the issue first thing in the morning

**Comment by aasimtaif** [comment]
the above error occurs when you try to upload a large  size image 
To resolve you have to increase the request size in the backend 
If I try to upload an image size of 170kb it gives me an error

**Comment by ojasaklechayt** [comment]
> the above error occurs when you try to upload a large size image To resolve you have to increase the request size in the backend If I try to upload an image size of 170kb it gives me an error

I see, can you make PR ? I will merge it.

**Comment by ojasaklechayt** [comment]
@aasimtaif Yes the code is working, actually the issue is that in the cors setting in the backend we have specified the frontend url and during the pr preview it uses different frontend url, hence giving the cors error. I feel it would be great if we can make the cors property general instread of specified for the main frontend url. Also I am closing the #32 pr since I have merged your latest pr, the changes of #32 pr is already been updated in the code.
